packages/terra-clinical-label-value-view/src/terra-dev-site/doc/example/LabelValueView.module.scss

```
@@ -5,4 +5,23 @@
     padding: 0 20px;
     width: 280px;
   }
+
+  .container {
+    display: flex;
+    font-size: 1rem;
+    line-height: 1.4285714285714286;
+    max-width: 100%;  // Needed for IE10 truncation
+    overflow: hidden; // VERY IMPORTANT FOR IE10
+  }
+
+  .icon {
+    flex: 0 0 auto;
+    overflow: hidden;
+    padding-right: 0.357142857142857rem;
+  }
+
+  .text {
+    color: #c00;
+    font-weight: 700; 
```

When we specify the "textStyle" as "attention", the styles of both the classes, i.e. `text` and `attention`, are applied.
So, you have to also add these styles : https://github.com/cerner/terra-clinical/pull/562/files#diff-ea4d170f02cea60d2157a80140ea9d5fR21-R22.
```suggestion
    font-weight: 700;
    flex: 1 1 auto;
    min-width: 10px; // Need min-width, as IE10 will completely squish display
```